src/components/Login/Logout.jsx

```diff
@@ -27,7 +28,22 @@ const Logout = () => {
         <label id="labelLogout" htmlFor="btnLogout">
           Click the following logout button to log out of your pod:{' '}
         </label>
-        <LogoutButton onLogout={handleLogout} />
+        <button type="submit" onClick={() => setShowConfirmation(true)}>
+          Logout
+        </button>
+        {showConfirmation ? (
+          <dialog open>
+            <p>Do you want to log out now?</p>
+            <div>
+              <LogoutButton onLogout={handleLogout}>
+                <button>Logout</button>
+              </LogoutButton>
+            </div>
+            <button onClick={() => setShowConfirmation(false)}>Cancel</button>
```

Comment:
Can move Line 42 inside the div with LogoutButton component:

```jsx
<div>
  <LogoutButton />
  <button>Cancel</button>
<div/>
```

Reply:
This has been moved.